ibis/expr/types/relations.py

```diff
@@ -3611,6 +3611,189 @@ def pivot_wider(
 
         return self.group_by(id_cols).aggregate(**aggs)
 
+    def relocate(
+        self,
+        *selectors: str | s.Selector,
+        before: str | s.Selector | None = None,
+        after: str | s.Selector | None = None,
+        **kwargs: str | s.Selector,
```

Comment:
How would a selector work here in kwargs? What's the meaning of:

```python
t.relocate(after="x", colname=s.of_type("float"))
```

Reply:
Yeah, that's definitely a mistake on my part. I'll remove the `s.Selector` annotation here.